packages/application-generic/src/usecases/process-subscriber/process-subscriber.usecase.ts

```diff
@@ -47,7 +47,11 @@ export class ProcessSubscriber {
       subscriberId: subscriberPayload.subscriberId,
     });
 
-    if (subscriber && !subscriberNeedUpdate(subscriber, subscriberPayload)) {
+    if (
+      subscriber &&
+      !subscriberNeedUpdate(subscriber, subscriberPayload) &&
+      !subscriberPayload.channels
```

Comment:
I don't entirely like this, but this is for now until we have a better diff logic happening here